python/tvm/relay/frontend/mxnet.py

```diff
@@ -1183,8 +1341,10 @@ def from_mxnet(symbol,
         params = {}
         for k, v in symbol.collect_params().items():
             params[k] = _nd.array(v.data().asnumpy())
-        data = mx.sym.Variable("data")
-        sym = symbol(data)
+        inputs = []
```

Comment:
These part should be fixed in the master now.

Reply:
I'll rebase.